src/style.css

```
@@ -0,0 +1,57 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+}
```

```suggestion
.stars {
  display: flex;
}
```
it's redundant 'cause you specified the width property in star styles